daemon/logger/fluentd/fluentd.go

```diff
@@ -285,12 +285,12 @@ func parseAddress(address string) (*location, error) {
 	}
 
 	switch addr.Scheme {
-	case "unix", "unixgram":
+	case "unix":
 		if strings.TrimLeft(addr.Path, "/") == "" {
 			return nil, errors.New("path is empty")
 		}
 		return &location{protocol: addr.Scheme, path: addr.Path}, nil
-	case "tcp", "tcp+tls", "udp":
+	case "tcp", "tls":
```

Comment:
Is "tcp+tls" equal to "tls"?  If so, should we convert "tcp+tls" to "tls" so existing configuration continues to work?  Or did "tcp+tls" never work at all?

Reply:
based on the vendor code, it looks like it never worked (I was initially considering the same) https://github.com/moby/moby/blob/cb176c848e0731f77fa48b4e1a90ae74d1f2deae/vendor/src/github.com/fluent/fluent-logger-golang/fluent/fluent.go#L243-L2610